libvcx/src/aries/handlers/proof_presentation/prover/prover.rs

```diff
@@ -15,6 +15,16 @@ pub struct Prover {
     prover_sm: ProverSM,
 }
 
+#[derive(Debug, PartialEq)]
+pub enum ProverState {
```

Comment:
This should also be moved to on `state_machine.rs` level

Reply:
dtto, `state_machine` is a private module for all SMs.